services/app-api/src/emailer/etaTemplates/sendWithdrawnRateStateEmail.eta

```diff
@@ -0,0 +1,20 @@
+<h1 style=<%~'"font-weight: normal;"'%>>Rate <%= it.rateName %> was withdrawn by CMS</h1>
+<b>Withdrawn by:</b> <%= it.withdrawnBy %><br />
+<b>Withdrawn on:</b> <%= it.withdrawnDate %><br />
+<b>Reason for withdrawal:</b> <%= it.withdrawnReason %><br />
+<b>Withdrawn from:</b>
+<% if (it.withdrawnFromContractData.length > 1) { %>
+<ul>
+<% it.withdrawnFromContractData.forEach(function(contract){ %>
+    <li>
+        <a href="<%= contract.submissionURL %>"><%= contract.contractName %></a>
```

Comment:
Nice to see us able to use this new contract field instead of needing to derive this